@kiva/kv-components/vue/KvLoanUse.vue

```diff
@@ -46,6 +46,10 @@ export default {
 			type: String,
 			default: '',
 		},
+		avoidLoanAmount: {
```

Comment:
What does this mean by "avoid"?

Reply:
Should this maybe be "hideLoanAmount". Is that what it's doing, removing it from the description?